ngraph/test/runtime/interpreter/evaluates_map.cpp

```diff
@@ -1529,6 +1529,33 @@ bool evaluate(const shared_ptr<op::v0::Elu>& op, const HostTensorVector& outputs
 template <element::Type_t ET>
 bool evaluate(const shared_ptr<op::v0::PriorBox>& op, const HostTensorVector& outputs, const HostTensorVector& inputs) {
     using T = typename element_type_traits<ET>::value_type;
+    op::v0::PriorBox::Attributes attrs = op->get_attrs();
```

Comment:
Looks like you don't need tp have these methods in evaluates map because both priorbox ops have evaluate methods

Reply:
removed these evaluates in commit abb47a7